src/Dfc.CourseDirectory.WebV2/BinaryStorageProvider/BlobStorageBinaryStorageProvider.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.IO;
+using System.Threading.Tasks;
+using Microsoft.Azure.Storage;
+using Microsoft.Azure.Storage.Auth;
+using Microsoft.Azure.Storage.Blob;
+
+namespace Dfc.CourseDirectory.WebV2.BinaryStorageProvider
+{
+    public class BlobStorageBinaryStorageProvider : IBinaryStorageProvider
+    {
+        private readonly CloudBlobClient _blobClient;
+        private readonly string _containerName;
+
+        public BlobStorageBinaryStorageProvider(
+            string accountName,
```

Comment:
I'm leaning these days towards passing primitive types in a container class or using the `Options` pattern.

Reply:
I have another patch that uses Options here but want to go over some config-related stuff in tech sessions first.